data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,62 @@ mission "Arachi Cartoonist 2"
 				accept
 			label noship
 			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`
+
+
+
+mission "Coalition Folklore: Arach"
+	minor
+	invisible
+	to offer
+		random < 30
+	source
+		attributes "arach" "folklore"
+	on offer
+		conversation
+			`Shortly after entering the spaceport, you observe an Arach in what you guess is a formal "dress," a delicate silk covering most of her body with each leg covered by its own embroidered skirt. She exclaims something loudly into a personal communication device before putting it away with a shove. Like everyone else who has momentarily been witness to this, you go to look away, but the Arach has cast their gaze around, seen you, and immediately begun to approach. "Our honorable human visitor, I salute," she says loudly as she closes the gap. "Say, Captain, appreciate the fine arts, do you? To a cultural soiree, I'm headed, but busy with work, dearest Papa is, so vacant, his reserved seat is. Accompany me, would you?"`
+			choice
+				`	"Is that some sort of event?"`
+				`	"I'm busy right now, so I'll pass."`
+					decline
+			`	"No simple gathering, but a collection of plays by esteemed professionals, completed by a dedicated orchestra, the soiree is," she explains. "A celebration of Arachi folklore, it is, which in our finest outfits, us local patrons attend. Streamed live for others, the show also is, but compare to the live performance, it cannot."`
+			choice
+				`	"Alright, I'm interested. Where is this 'soiree' taking place?"`
+				`	"I'm not a fan of plays, sorry."`
+					decline
+			`	You follow her around the central portion of the city to a building, which from the outside looks like some form of auditorium; there, some Arachi in uniform are seeing to the entries of the "patrons" of the event. "A human? Quite an exotic guest you've brought, lady Matlub," one of them says to your guide as she shows him a touchpad displaying two tickets. He verifies them and allows you two to enter. The inside reminds you of a human opera house, with a large stage in front of the dozens of rows of seats, and six levels of balconies on both sides. Matlub leads you to two excellent seats in one of the first few rows, and she helps you "adjust" the chair so that you're comfortably seated. Just before the stage is an area a few steps below the level of the seats, with a few Arachi musicians fine-tuning a variety of instruments. Up on the ceiling, and on all of the balconies, you see small cameras pointed at the stage to broadcast the event for those who didn't have the chance to attend it live.`
```

Comment:
```suggestion
			`	You follow her around the central portion of the city to a building which from the outside looks like some form of auditorium; there, some Arachi in uniform are seeing to the entries of the "patrons" of the event. "A human? Quite an exotic guest you've brought, lady Matlub," one of them says to your guide as she shows him a touchpad displaying two tickets. He verifies them and allows the two of you to enter. The inside reminds you of a human opera house, with a large stage in front of dozens of rows of seats, and six levels of balconies on both sides. Matlub leads you to two excellent seats in one of the first few rows, and she helps you "adjust" the chair so that you're more comfortably seated. Just before the stage is an area a few steps below the level of the seats, with a few Arachi musicians fine-tuning a variety of instruments. Up on the ceiling, and on all of the balconies, you see small cameras pointed at the stage to broadcast the event for those who didn't have the chance to attend it live.`
```